scripts/update/0.6-0.6.1/cluster.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -euo pipefail
+
+mode="${1}"
+
+function run_on_host() {
+  nsenter -a -t 1 /bin/sh -c "${1}"
+}
+
+function update_etcd() {
+  if [ "${mode}" != "controller" ]; then
+    echo "Nothing to do. Not a controller node."
+    return
+  fi
+
+  rkt_etcd_cfg="/etc/systemd/system/etcd-member.service.d/40-etcd-cluster.conf"
+  docker_etcd_cfg="/etc/kubernetes/etcd.env"
+  docker_etcd_svc="/etc/systemd/system/etcd.service"
+
+  if [ -f "${rkt_etcd_cfg}" ]; then
+    echo "Nothing to do. Rkt based etcd node."
+    return
+  fi
+
+  if grep "^IMAGE_TAG" "${docker_etcd_cfg}" >/dev/null; then
+    echo "etcd env var file ${docker_etcd_cfg} is already updated."
+    return
+  fi
+
+  echo -e "\nUpdating etcd file...\nOld etcd config file:\n"
+  cat "${docker_etcd_cfg}"
+  sed 's|ETCD_IMAGE_TAG|IMAGE_TAG|g; s|ETCD_IMAGE_URL|IMAGE_URL|g; s|ETCD_SSL_DIR|SSL_DIR|g; s|ETCD_USER|USER|g' ${docker_etcd_cfg} >/tmp/etcd.env
```

Comment:
I'm wondering if this is worth having an upgrade script, as it brings no functional changes.

Reply:
@surajssd 

What do you think about the need for a script here. Is this something we can do without ?

I personally think that we should reduce the manual steps every release.

Reply:
I think any host related changes should go with the release bundle.

If we don't do these incremental changes the skew between what is shipped and what exists on the prod cluster increases, may lead to control plan recycle which is not desirable.